### ISSUE TITLE
NCBI Ingest: Correct geolocation metadata mappings

### ISSUE THREAD

**Issue by anna-parker** [loculus-project/loculus#2788]
- [ ] #3105
![image](https://github.com/user-attachments/assets/ce55f719-bad7-46ab-b5ac-c631ec71003d)

- [x] Additionally we map ncbiGeoRegion we map to geoLocAdmin2, which is not correct as ncbiGeoRegion corresponds to continent and geoLocAdmin2 should be "A local administrative region from which the sample was collected (ex: county or municipality)": 
![image](https://github.com/user-attachments/assets/f9cc6ec4-09ed-42a2-aa16-9988a3945aaa)

- [ ] Can we reconsider the displayName for these fields? I think it will be confusing to submitters that geoLocAdmin1 gets mapped to collection subdivision etc.

**Comment by corneliusroemer** [comment]
Yeah this was on my list of things to fix/improve eventually. Thanks for making an issue.
